cmd/cmd_account.go

```diff
@@ -2,243 +2,92 @@ package main
 
 import (
 	"context"
+	"encoding/json"
+	"errors"
 	"fmt"
-	"strings"
+	"os"
+	"path/filepath"
 
 	"cosmossdk.io/math"
+	sdktypes "github.com/bnb-chain/greenfield-go-sdk/types"
 	"github.com/bnb-chain/greenfield/sdk/types"
-	gnfdsdktypes "github.com/bnb-chain/greenfield/sdk/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/urfave/cli/v2"
 )
 
-// cmdCreatePaymentAccount creates a payment account under the owner
-func cmdCreatePaymentAccount() *cli.Command {
+// cmdImportAccount import the account by private key file
+func cmdImportAccount() *cli.Command {
 	return &cli.Command{
-		Name:      "create",
-		Action:    CreatePaymentAccount,
-		Usage:     "create a payment account",
-		ArgsUsage: "",
+		Name:      "import",
+		Action:    importKey,
+		Usage:     "import the account by the private key file",
+		ArgsUsage: " <privateKeyFile>",
 		Description: `
-Create a payment account
+Import account info from private key file and generate a keystore file to manage user's private key information.
+If no keyfile is specified by --keystore or -k flag, a keystore will be generated at the default path （homedir/.gnfd-cmd/keystore/key.json）
+Users need to set the private key file path which contain the origin private hex string .
 
 Examples:
-# Create a payment account
-$ gnfd-cmd payment-account create`,
-	}
-}
-
-func CreatePaymentAccount(ctx *cli.Context) error {
-	client, err := NewClient(ctx)
-	if err != nil {
-		return toCmdErr(err)
-	}
-	c, createPaymentAccount := context.WithCancel(globalContext)
-	defer createPaymentAccount()
-	acc, err := client.GetDefaultAccount()
-	if err != nil {
-		return toCmdErr(err)
+// key.txt contains the origin private hex string 
+$ gnfd-cmd  -k key.json  account import  key.txt `,
```

Comment:
move -k key.json behind the import key.txt is better?

Reply:
This is a global flag, which does not belong to the account command. All other commands share this flag, so needs to put it in front.

Reply:
If put it in the back,  we need to add a new flag which just belong to the "account import "command